src/components/PageBlocks/RecordListFilter.vue

```diff
@@ -0,0 +1,249 @@
+<template>
+  <div>
+    <b-btn
+      variant="link p-0 ml-1"
+      @click="onOpen"
+      v-b-tooltip.focus
+      :id="selectedFieldName"
+      class="btn-filter"
+      v-bind:class="{ 'text-primary': inFilter }"
+    >
+      <font-awesome-icon :icon="['fas', 'filter']" />
+    </b-btn>
+    <b-tooltip
+      custom-class="ctooltip"
+      ref="tooltip"
+      class="ctooltip1"
+      :target="selectedFieldName"
+      triggers="focus"
+    >
+      <div class="filter-tooltip">
+        <section
+          class="filter-row"
+          v-for="(filter, index) in componentFilter"
+          :key="index"
+        >
+          <label v-if="index === 0">Where</label>
+          <b-form-select
+            v-else
+            v-model="filter.condition"
+            :options="conditions"
+            size="md"
+          ></b-form-select>
+          <b-form-select
+            v-model="filter.name"
+            :options="fields"
+            value-field="key"
+            text-field="label"
+            size="md"
+            @change="onChange(filter.name, index)"
+          ></b-form-select>
+          <b-form-select
+            v-model="filter.equation"
+            :options="equations"
+            size="md"
+          ></b-form-select>
+
+          <!-- Multi field type, String,Number,Bool etc -->
+          <b-form-input
+            v-if="filter.kind === 'String'"
+            v-model="filter.query"
+            placeholder="Enter value"
+            size="md"
+          ></b-form-input>
+          <b-form-input
+            type="number"
+            v-if="filter.kind === 'Number'"
+            v-model="filter.query"
+            placeholder="Enter value"
+            size="md"
+          ></b-form-input>
+          <b-form-select
+            v-if="filter.kind === 'Bool'"
+            v-model="filter.query"
+            :options="checkbox"
+            size="md"
+          ></b-form-select>
+          <b-form-select
+            v-if="filter.kind === 'Select'"
+            v-model="filter.query"
+            :options="filter.options"
+            size="md"
+          ></b-form-select>
+
+          <b-button
+            @click="deleteFilter(index)"
+            variant="link"
+            class="align-baseline p-0 text-dark"
+          >
+            <font-awesome-icon :icon="['far', 'trash-alt']" />
+          </b-button>
+        </section>
+        <b-btn
+          variant="link"
+          class="align-baseline p-0 add-filter"
+          @click="addFilter"
+        >
+          + Add filter
+        </b-btn>
+        <footer class="filter-footer">
+          <b-btn @click="save" variant="primary" size="md">
+            Save filter
```

Comment:
All static strings will need to be turned into translation string, I usually do this at the end of a feature. Since then you can see what strings you need. We will discuss how to do this on our sync

Reply:
Also no need for the icon, just save should be enough